app/src/ui/about/about.tsx

```diff
@@ -194,6 +194,10 @@ export class About extends React.Component<IAboutProps, IAboutState> {
       )
     }
 
+    if (__LINUX__) {
```

Comment:
Minor nit: since this branch can return `null` I think it should be put at the top of the function.